_posts/2022-02-16-cockpit-263.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Cockpit 263
+author: garrett
+date: '2022-02-16'
+tags: cockpit
+slug: cockpit-263
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 263:
+
+
+
+## The back and forward buttons in your browser now work as expected
+
+Cockpit used to interfere with the browser history, and hitting the back button would frequently get into a loop, for example. This has been fixed and the back and forward button are now enabled again in Cockpit Client.
```

Comment:
```suggestion
Cockpit used to interfere with browser history. Clicking the back button would frequently get into a loop. Browser history handling is now fixed, so navigating back works as expected.

 Additionally, back and forward buttons have been added to [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient), the desktop app to connect to Cockpit servers using SSH.
```

Reply:
I've reworded a bit, added a link to Cockpit Client, and explained what it is (as not everyone knows).

Reply:
We could add a screenshot of Cockpit Client's back/forward buttons here. But we should definitely at least update Cockpit Client's screenshots on Flathub when we can.